File: StraviaTec-Web/Models/PatrocinadorEvento.cs
using System;
using System.Collections.Generic;

namespace StraviaTec_Web.Models
{
    public partial class PatrocinadorEvento
    {
        public int Id { get; set; }
        public int IdEvento { get; set; }
        public int IdPatrocinador { get; set; }

        public virtual Evento IdEventoNavigation { get; set; }
        public virtual Patrocinador IdPatrocinadorNavigation { get; set; }
    }
}

File: StraviaTec-Web/ClientApp/src/app/models/patrocinadorEvento.ts
export class PatrocinadorEvento {
    id: number;
    idEvento: number;
    idPatrocinador: number;
    nombreComercial: string;

    constructor(
        id: number,
        idEvento: number,
        idPatrocinador: number) {
            this.id = id;
            this.idEvento = idEvento;
            this.idPatrocinador = idPatrocinador;
        }
}